src/Datepicker.vue

```diff
@@ -310,6 +315,11 @@ export default {
       this.$emit('input', new Date(timestamp))
     },
 
+    clearDate () {
+      this.selectedDate = ''
+      this.$emit('selected', this.selectedDate)
```

Comment:
I love this :) i hope they merge it, can i suggest also emitting a "cleared" event? just an idea, in case is needed to know if the date was cleared or not